src/components/CreateTask.js

```diff
@@ -0,0 +1,109 @@
+/** @jsx jsx */
+import { jsx } from 'theme-ui';
+import { Input, Label, Button } from '@theme-ui/components';
+import { Formik, Form } from 'formik';
+import { axiosWithAuth } from '../utils/axiosWithAuth';
+
+const CreateTask = () => {
+  const initialValues = {
+      task_name: "",
+      category_name: "",
+      due_date: "",
+      all_day: "",
+      task_notes: "",
+      is_complete: ""
+    }
+
+
+
+  const handleSubmit = values => {
```

Comment:
Since creating a new task is going to be causing the state with the user's tasks to update, we should probably move the axios call out into the TaskContext and TaskProvider